src/components/CardElementPic.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const PicWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  background: white;
+  margin: 10px 20px;
+  width: 40%;
+`;
+
+const StyledImageDiv = styled.div`
+  height: 90vh;
+`;
+
+const Details = styled.p`
+  color: #4ecbe0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 8pt;
```

Comment:
Minimal Font-size should be 16px for "barriere-freiheit"

Reply:
true that. i have to change the style a bit, therefore its not relevant anymore! But good eyes you have ;)

Reply:
i changed my mind. chose 16pt instead of 8pt